phylogenetic/defaults/color_orderings.tsv

```diff
@@ -225,38 +225,11 @@ region	North America
 
 ################
 
-
-recency	Older
-recency	One month ago
-recency	One week ago
-recency	3-7 days ago
-recency	1-2 days ago
-recency	New
-
-################
-
-serotype_genbank	denv1
-serotype_genbank	denv2
-serotype_genbank	denv3
-serotype_genbank	denv4
-
-genotype_nextclade	DENV1/I
-genotype_nextclade	DENV1/II
-genotype_nextclade	DENV1/III
-genotype_nextclade	DENV1/IV
-genotype_nextclade	DENV1/V
-genotype_nextclade	DENV2/AA
-genotype_nextclade	DENV2/AI
-genotype_nextclade	DENV2/AII
-genotype_nextclade	DENV2/AM
-genotype_nextclade	DENV2/C
-genotype_nextclade	DENV2/S
-genotype_nextclade	DENV3/I
-genotype_nextclade	DENV3/II
-genotype_nextclade	DENV3/III
-genotype_nextclade	DENV3/IV
-genotype_nextclade	DENV4/I
-genotype_nextclade	DENV4/II
-genotype_nextclade	DENV4/S
-
-################
+# genotype assigned by Nextclade
+clade_membership	Clade I
+clade_membership	Clade II
+clade_membership	Clade III
+clade_membership	Clade IV
+clade_membership	Clade V
+clade_membership	Clade VI
+clade_membership	Clade VII
```

Comment:
Hmm, the clade order in the legend is not matching the order here

![Screenshot 2024-11-25 at 11 40 35 AM](https://github.com/user-attachments/assets/8ac1f253-f56d-4831-9644-1247d71fd992)

Edit: OH, this needs to be `clade` instead of `clade_membership`.

Reply:
good eyes!